### PR TITLE
Contact-Page

### PR DESCRIPTION
**Pull request by waqaskanju**
In this pull we implemented these task:
- Created Contact Us Page
- Created Navigation Menu
- Added Styling to the project
- Removed Linter Errors.

### CODE REVIEW

File: html.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;700&display=swap');
```
Comment (elmar8287) [inline html.css:1]: - [ ] Please, consider following general requirements:
  - Pull Request should have a descriptive title telling you what changes are inside.
  - Pull Request should have a short summary describing introduced changes in more detail.
Screen Shot 2022-05-26 at 09 58 39

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

I want to highlight:

- Nice design 👍 
- No linter issues

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- Please, consider future giving titles for commits with Capital letters, it will make your commits history more professional 👍 
- Also, please, try to give commits a more descriptive title - short info on what was committed 👍 - that will also make your job professional

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: html.css (hunk 1 above)
Comment (waqaskanju) [reply html.css:1]: image
Thanks, Done the changes.

Comment (elmar8287) [reply html.css:1]: 👍

**Pull request merged**